pkg/gatherers/clusterconfig/active_alerts.go

```diff
@@ -51,17 +65,20 @@ func gatherActiveAlerts(ctx context.Context, alertsClient rest.Interface) ([]rec
 		return nil, []error{err}
 	}
 
+	// the maximal number of recorded alerts
+	var limit = 1000
```

Comment:
This used to be a constant declared at the top of the file. IMHO, it seems a little bit problematic to have it declared in the package scoped and is used only here, so, I converted it to a local variable inside the method.